requirements_frozen.txt

```diff
@@ -1,26 +1,21 @@
 autorepr==0.3.0
-certifi==2021.5.30
-charset-normalizer==2.0.6
-codecheck==1.1.3
+certifi==2023.7.22
+charset-normalizer==3.2.0
+codecheck==1.3.1
 compiler-identification==1.0.3
-docker==5.0.2
+docker==6.1.3
 downloadutil==1.0.2
-idna==3.2
-llvm-installer==1.0.2
-mypy-extensions==0.4.3
-mypy==0.910
-nodeenv==1.7.0
-packaging==21.0
-pycodestyle==2.7.0
-pyparsing==2.4.7
-pyright==1.1.292
-requests==2.26.0
+idna==3.4
+llvm-installer==1.3.6
```

Comment:
Let's avoid updating llvm-installer for now. This will cause compiler minor version upgrades.